From 929e7671178220e6ef29ea91e026fdcfb33d5d6b Mon Sep 17 00:00:00 2001
From: Ian Campbell <ian.campbell@citrix.com>
Date: Fri, 8 Oct 2010 17:11:24 +0100
Subject: [PATCH 16/26] xen: correct parameter type for pirq_eoi

Fixes:
  CC      drivers/xen/events.o
/local/scratch/ianc/devel/kernels/linux-2.6/drivers/xen/events.c:1452: warning: initialization from incompatible pointer type

Signed-off-by: Ian Campbell <ian.campbell@citrix>
Signed-off-by: Jeremy Fitzhardinge <jeremy.fitzhardinge@citrix.com>
(cherry picked from commit 7c42097171f2e0beafa16e007a06e464b3014bea)

Signed-off-by: Ian Campbell <ijc@hellion.org.uk>
---
 drivers/xen/events.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/xen/events.c b/drivers/xen/events.c
index 1941384..1dd1f29 100644
--- a/drivers/xen/events.c
+++ b/drivers/xen/events.c
@@ -436,7 +436,7 @@ static bool identity_mapped_irq(unsigned irq)
 	return irq < get_nr_hw_irqs();
 }
 
-static void pirq_eoi(int irq)
+static void pirq_eoi(unsigned int irq)
 {
 	struct irq_info *info = info_for_irq(irq);
 	struct physdev_eoi eoi = { .irq = info->u.pirq.gsi };
-- 
1.7.5.4

